assets/dev/js/editor/elements/manager.js

```diff
@@ -69,5 +69,11 @@ export default class ElementsManager {
 
 			this.registerElementType( element );
 		} );
+
+		if ( elementorCommon.config.experimentalFeatures.container ) {
+			const ContainerClass = require( './types/container' ).default;
+
+			this.registerElementType( new ContainerClass() );
+		}
```

Comment:
Move to container PR

Reply:
Cannot `feature/container` does not include `infra/nested`

Reply:
Remove, Should ve included in `infra/nested`.